client/layout/guided-tours/config.js

```diff
@@ -118,10 +120,122 @@ const tours = {
 	},
 	themes: {
 		meta: {
-			version: '20160719',
+			version: '20160609',
 			path: '/design',
-			// don't enable this in production (yet)
-			context: () => 'production' !== config( 'env' ),
+			context: ( state ) => config.isEnabled( 'guided-tours/themes' ) && isNewUser( state ),
+		},
+		description: 'Learn how to find and activate a theme',
+		showInContext: state => getSectionName( state ) === 'themes',
+		init: {
+			text: 'Hey there! Want me to show you how to find a great theme for your site?',
+			type: 'FirstStep',
+			placement: 'right',
+			next: isMobile() ? 'mobileSearch' : 'search',
+		},
+		mobileSearch: {
+			text: 'Click the search button to search for themes',
+			type: 'ActionStep',
+			target: '.themes__search-card .search-open__icon',
+			placement: 'below',
+			arrow: 'top-left',
+			next: 'search',
+		},
+		search: {
+			text: 'Search for a specific theme name or feature here. Try typing something — for example, "business".',
+			type: 'ActionStep',
+			target: '.themes__search-card .search-open__icon',
+			placement: 'below',
+			continueIf: state => {
+				const params = getQueryParams( state );
+				return params && params.search && params.search.length && ! isFetchingNextPage( state ) && getThemesList( state ).length > 0;
+			},
+			arrow: 'top-left',
+			next: isMobile() ? 'mobileCloseSearch' : 'filter',
+		},
+		mobileCloseSearch: {
+			text: 'Close the search',
+			type: 'ActionStep',
+			target: '.themes__search-card .search-close__icon',
+			placement: 'below',
+			arrow: 'top-left',
+			next: 'mobileFilter',
+		},
+		filter: {
+			text: 'Here you can filter between free and premium themes. Try filtering by _free_ themes now.',
+			type: 'ActionStep',
+			target: 'themes-tier-dropdown',
+			placement: 'above',
+			continueIf: state => {
+				const params = getQueryParams( state );
+				return params && params.tier === 'free';
+			},
+			arrow: 'bottom-right',
+			next: 'choose-theme',
+		},
+		mobileFilter: {
+			text: 'Here you can filter between free and premium themes. Try filtering by _free_ themes now.',
+			type: 'ActionStep',
+			target: '.themes__search-card .section-nav__mobile-header',
+			placement: 'above',
+			showInContext: () => isMobile(),
```

Comment:
Yeah, I'm not opposed. However, it'll be nice to have something in mobile soon enough so that we can tackle the expectable oddities/jankiness early in development.